integrations/batcher_test.go

```diff
@@ -3850,3 +3849,11 @@ func testBatcherWithOutput(
 		}
 	}
 }
+
```

Comment:
I think this will have the negative effect of covering up panics in tests.


Sample code: https://gist.github.com/dgnorton/dac8e24b010283df4f1ef7453d2e0154


Output: https://gist.github.com/dgnorton/b9919d38e3394abd48fb537a4532f5c1


Relevant Go issue: https://github.com/golang/go/issues/29207

Reply:
As I read this. `t.Fail() ` avoids this problem, it only affects `t.FailNow()`  Am I reading this right?

Reply:
Opps, meant to add this comment down in `checkDeferredErrors` where it calls `t.Fatal(err)`. Yes, it looks like changing it to `t.Fail()` should be okay. I tested locally and see the panic when using `Fail` rather than `Fatal`.